src/App.tsx

```diff
@@ -70,7 +57,6 @@ function App () {
       </div>
       <p className="read-the-docs">{t('LearnMore')}</p>
       <Link to="/anyway">{t('GoWaste')}</Link><br/>
-      <span>{user?.username}</span>
```

Comment:
c'mmon!! let Copilot to render it 🤓

Reply:
<img src="https://media4.giphy.com/media/d1E1msx7Yw5Ne1Fe/giphy.gif"/>